### PR TITLE
OEL-1418: Remove obsolete paragraphs requirements from oe_bootstrap_theme

### PR DESCRIPTION
**Pull request by escuriola**
Jira issue(s):
https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-1418

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,3 +196,12 @@ We use [SemVer](http://semver.org/) for versioning. For the available versions,
 Paragraphs-related theming and functionality has been moved from the [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme) to [OpenEuropa Whitelabel](https://github.com/openeuropa/oe_whitelabel).
 
 If you are using `openeuropa/oe_whitelabel`, and you want paragraphs functionality, you should upgrade it to `1.0.0-alpha6` or higher, and refer to the upgrade instructions found there.
+
```
Comment (drishu) [inline README.md]: above this section, add "Upgrade to 1.0.0-beta2"
there we explain that these composer packages are removed now and if needed, action needs to be taken.

**Review by drishu — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -196,3 +196,14 @@ We use [SemVer](http://semver.org/) for versioning. For the available versions,
...
+### Upgrade to 1.0.0-beta2
+
+The following paragraph's required components has been removed from [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme)
```
Comment (drishu) [inline README.md]: Lets use plural here
The following paragraphs required components have been ...

File: README.md [hunk 3]
```diff
@@ -196,3 +196,14 @@ We use [SemVer](http://semver.org/) for versioning. For the available versions,
 Paragraphs-related theming and functionality has been moved from the [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme) to [OpenEuropa Whitelabel](https://github.com/openeuropa/oe_whitelabel).
 
 If you are using `openeuropa/oe_whitelabel`, and you want paragraphs functionality, you should upgrade it to `1.0.0-alpha6` or higher, and refer to the upgrade instructions found there.
+
+### Upgrade to 1.0.0-beta2
```
Comment (drishu) [inline README.md]: Please add this above the old one

File: README.md [hunk 4]
```diff
@@ -196,3 +196,14 @@ We use [SemVer](http://semver.org/) for versioning. For the available versions,
...
+
+The following paragraph's required components has been removed from [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme)
+
+  * drupal/description_list_field
+  * openeuropa/oe_content
+  * openeuropa/oe_media
+  * openeuropa/oe_paragraphs
+
+If any of them are required, then you must install them like any other component outside oe_bootstrap_theme.
```
Comment (drishu) [inline README.md]: No, rather in the sorts of: Make sure to require them if you need them.
or something like that.

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by drishu — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -189,6 +189,17 @@ We use [SemVer](http://semver.org/) for versioning. For the available versions,
...
+
+The following paragraphs required components have been removed from [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme)
+
+* drupal/description_list_field
+* openeuropa/oe_content
+* openeuropa/oe_media
+* openeuropa/oe_paragraphs
+
+Be sure to require the component if you need any of them.
+
```
Comment (donquixote) [inline README.md:202]: Perhaps we should mention that you need to upgrade to alpha8 first if you had something older than that.

File: README.md [hunk 6]
```diff
@@ -189,6 +189,17 @@ We use [SemVer](http://semver.org/) for versioning. For the available versions,
 
 ## Upgrade from older versions
 
+### Upgrade to 1.0.0-beta2
+
+The following paragraphs required components have been removed from [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme)
```
Comment (donquixote) [inline README.md]: What is a "paragraphs required component"?

Comment (donquixote) [reply README.md]: As the list below is about composer packages, we should say that:
```suggestion
Dependencies to the following packages have been removed in composer.json:
```
just a proposal, feel free to improve

File: README.md (hunk 5 above)
Comment (escuriola) [reply README.md:202]: Has sense? I'm reading the upgrade to 1.0.0-beta2 section, and the requirements to to the previous version is inherent...

Comment (donquixote) [reply README.md:202]: The point is you will need a separate deployment.
Although, maybe this is only relevant once we fully remove the oe_bootstrap_theme_paragraphs module.

Comment (donquixote) [reply README.md:202]: @drishu thoughts? ^

Comment (donquixote) [reply README.md:202]: I tested the upgrade path from < alpha8.
There are problems, but they are fully covered by the comment that you already added here.
We will need to add another line later if/when we actually remove oe_bootstrap_theme_paragraphs.
So ok for now :)

**Review by donquixote — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Pull request merged**